CHANGELOG.md

```diff
@@ -5,6 +5,32 @@ information on the list of deprecated flags and APIs please have a look at
 https://docs.docker.com/engine/deprecated/ where target removal dates can also
 be found.
 
+## 17.06.1-ce (2017-07-XX)
+
+* (build) Fix add from remote url [#89](https://github.com/docker/docker-ce/pull/89)
+* Prevent a goroutine leak when healthcheck gets stopped [#90](https://github.com/docker/docker-ce/pull/90)
+* Make plugin removes more resilient to failure [#91](https://github.com/docker/docker-ce/pull/91)
+* Do not error on relabel when relabel not supported [#92](https://github.com/docker/docker-ce/pull/92)
+* Limit max backoff delay to 2 seconds for GRPC connection [#94](https://github.com/docker/docker-ce/pull/94)
+* Fix stderr logging for journald and syslog [#95](https://github.com/docker/docker-ce/pull/95)
+* Use man-pages and docs from docker/cli instead of moby/moby [#96](https://github.com/docker/docker-ce/pull/96)
+* Show progress of replicated tasks before they are assigned [#97](https://github.com/docker/docker-ce/pull/97)
+* Fix log readers can block writes indefinitely [#98](https://github.com/docker/docker-ce/pull/98)
+* (middleware) Redact secret data on "secret create" [#99](https://github.com/docker/docker-ce/pull/99)
+* Fix handling of remote "git@" notation [#100](https://github.com/docker/docker-ce/pull/100)
+* (engine) Vendor forks of containerd + runc + runtime-spec [#102](https://github.com/docker/docker-ce/pull/102)
+* Various completion script updates [#104](https://github.com/docker/docker-ce/pull/104)
+* Various docs fixes [#105](https://github.com/docker/docker-ce/pull/105)
+* Docker wait hangs indefinitely for non-existent container (vendor) [#106](https://github.com/docker/docker-ce/pull/106)
+* Do not add duplicate platform information to service spec [#107](https://github.com/docker/docker-ce/pull/107)
+* Stop trying to load images on an incompatible OS [#108](https://github.com/docker/docker-ce/pull/108)
```

Comment:
would this work? "Improve error message if an image for an incompatible OS is loaded"